lib/octokit/client/users.rb

```diff
@@ -309,9 +307,9 @@ def emails(options = {})
       # @see https://developer.github.com/v3/users/emails/#add-email-addresses
       # @example
       #   @client.add_email('new_email@user.com')
-      def add_email(email, options = {})
+      def add_email(email, _options = {})
```

Comment:
It's pretty weird that this (and other) methods take this `options` argument but don't use it at all. I wonder if it's intentional. (It would be a breaking change to remove them.)